test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -561,6 +561,44 @@ foo
             }
         }
     }
+
+    Context "-WorkingDirectory parameter" {
+        BeforeAll {
+            $folderName = (New-Guid).ToString() + " test";
+            New-Item -Path ~/$folderName -ItemType Directory
```

Comment:
Maybe use `$pwd` instead of `~` alias? Below too.

Reply:
I wonder - is this a suitable place for `~` tests?

Reply:
The intent is to validate a path understood by the FileSystemProvider which abstracts `~` for Windows and Unix

Reply:
I don't understand why we place FileSystemProvider tests in ConsoleHost.Tests.ps1.

Reply:
The documentation for this parameter is that any file system provider path is accepted, and `~` is handled by the provider which is why I added that test case.